src/Netflix/Netflix.ts

```diff
@@ -360,7 +357,7 @@ export default <IPluginBase & IPlugin>{
       },
       {
         name: "Change Audio",
-        match: ["[change/switch] audio to *", "audio to *"],
+        match: ["[/change/switch] audio to *"],
```

Comment:
this will create a match string with a leading space. You should include the space like this: "[/change /switch ]audio to *"

Reply:
(other match strings below also need to be adjusted for this leading space)

Reply:
I see.
Saying "audio to *" works though with this.